Jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env groovy
+
+library("govuk")
+
+node("postgresql-9.3") {
+  govuk.buildProject(
+    beforeTest: {
+      govuk.setEnvar("TEST_COVERAGE", "true")
+    }
```

Comment:
What about adding testing with brakeman here:
https://github.com/alphagov/content-publisher/blob/7ce30f12d9878def9254407bf0c707b07af4b810/Jenkinsfile#L13

Reply:
This is the default for a rails app. as documented here:  https://github.com/alphagov/govuk-jenkinslib - they are running - you can see output in Jenkins.

Reply:
Ah, looks like it's only recently default to `true` https://github.com/alphagov/govuk-jenkinslib/pull/19